mesonbuild/mdist.py

```diff
@@ -158,6 +159,9 @@ def check_dist(packagename, meson_command, privdir):
         unpacked_files = glob(os.path.join(unpackdir, '*'))
         assert(len(unpacked_files) == 1)
         unpacked_src_dir = unpacked_files[0]
+        with open(os.path.join(bld_root, 'meson-info', 'intro-buildoptions.json')) as boptions:
+            meson_command += ['-D{name}={value}'.format(**o) for o in json.load(boptions)
+                if o['name'] not in ['backend', 'install_umask']]
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5738#issue-4396808))

<sub>posted by Sider</sub>